source/install.txt

```diff
@@ -154,6 +154,23 @@ Install Compass
             to modify your system settings to grant |compass-short|
             permissions to run. You may be prompted to enter your
             system password before launching |compass-short|.
+         
+         
+         7. Allow macOS to trust |compass-short|. If you receive a
+            security error when starting |compass-short| indicating that
+            the developer could not be identified or verified, perform the
+            following actions to allow |compass-short| to run:
+            
+            a. Open *System Preferences*.
+   
+            #. Select the *Security and Privacy* pane.
+
+            #. Under the *General* tab, click the button to the right of the
```

Comment:
After this step, I think they may need to re-run Compass in this case, so possibly add a step:

`d. If necessary, re-open |compass-short|.`